### PR TITLE
fix: add missing schema so eslint would accept options

### PR DESCRIPTION
**Pull request by samtsai**

### CODE REVIEW

File: src/rules/alias.test.ts [hunk 1]
```diff
@@ -57,6 +57,13 @@ tester.run('paths-alias', rule, {
 			errors: ['Update import to @foo/x/y/z'],
 			output: `import z from '@foo/x/y/z';`,
 		},
+		{
+			name: 'relative named import from alias must be fixed',
+			filename: path.resolve('./src/nested/index.ts'),
+			code: `import { x } from '../qux/x/y/z';`,
+			errors: ['Update import to @qux/x/y/z'],
+			output: `import { x } from '@qux/x/y/z';`,
+		},
```
Comment (vitonsky) [inline src/rules/alias.test.ts:66]: What difference of test that described below?

```
		{
			name: 'relative import from alias used in another alias must be fixed',
			filename: path.resolve('./src/foo/index.ts'),
			code: `import z from '../bar/x/y/z';`,
			errors: ['Update import to @bar/x/y/z'],
			output: `import z from '@bar/x/y/z';`,
		},
```

Could you elaborate for what case this test for

File: src/rules/alias.ts [hunk 2]
```diff
@@ -49,8 +49,7 @@ function findAlias(
 		// Remove basedir and slash in start
 		const slicedImportPath = importPath
 			.slice(baseDir.length + 1)
-			.slice(path.dirname(matchedPath).length + 1);
-
+			.slice(path.dirname(path.normalize(matchedPath)).length + 1);
```
Comment (vitonsky) [inline src/rules/alias.ts:52]: Why this change for? What problem (maybe potential case?) we have with current code?

**Review by vitonsky — COMMENTED** [review]
Thank you for your patch.

Let's clarify about changes not related to schema definition, and will merge.

File: src/rules/alias.ts (hunk 2 above)
Comment (samtsai) [reply src/rules/alias.ts:52]: Without normalizing, if you have a tsconfig path like:
`"@e2e/*": ["./cypress/e2e/*"],`

The slicing logic will return the string missing two characters, due to the additional two characters from `./`.

So the message I received was something like: `Update import to @e2e/st` instead of `Update import to @e2e/test`

File: src/rules/alias.test.ts (hunk 1 above)
Comment (samtsai) [reply src/rules/alias.test.ts:66]: Yea so this one was trickier to explain but it's really testing the `tsconfig` alias path more than the import itself. It is testing the change below.

Comment (vitonsky) [reply src/rules/alias.test.ts:66]: I can verify that this test is useful and can catch a problem if revert fix below

image
